jkube-kit/doc/src/main/asciidoc/inc/build/_buildargs.adoc

```diff
@@ -2,16 +2,31 @@
 As described in section <<build-configuration,Configuration>> for external Dockerfiles https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables-build-arg[Docker build arg] can be used. In addition to the
 configuration within the plugin configuration you can also use properties to specify them:
 
-* Set a system property when running Maven, eg.: `-Ddocker.buildArg.http_proxy=http://proxy:8001`. This is especially
+ifeval::["{plugin-type}" == "gradle"]
+* Set a system property when running {plugin-type}, e.g: `-Pdocker.buildArg.http_proxy=http://proxy:8001`. This is especially
 useful when using predefined Docker arguments for setting proxies transparently.
-* Set a project property within the `pom.xml`, eg.:
+* Set a project property within the `build.gradle`, e.g:
+
+.Example
+[source,properties,indent=0,subs="verbatim,quotes,attributes"]
+----
+docker.buildArg.myBuildArg = myValue
+----
+endif::[]
+ifeval::["{plugin-type}" == "maven"]
+* Set a system property when running {plugin-type}, e.g: `-Ddocker.buildArg.http_proxy=http://proxy:8001`. This is especially
+useful when using predefined Docker arguments for setting proxies transparently.
```

Comment:
Minor: these 2 lines are common to maven and gradle, right? could these be out of the ifeval blocks?

Reply:
Thanks for noticing! Let me update it